### PR TITLE
Fix depset issues with/upgrade to Bazel 0.27.0

### PR DESCRIPTION
**Pull request by factuno-db**
This PR updates rules_closures and Bazel.

`depset` is no longer iterable starting from Bazel 0.27.0.

### CODE REVIEW

File: bazel/closure_grpc_web_library.bzl [hunk 1]
```diff
@@ -55,12 +55,12 @@ def _generate_closure_grpc_web_srcs(
...
         sources,
         transitive_sources):
-    all_sources = [src for src in sources] + [src for src in transitive_sources]
+    all_sources = [src for src in sources] + [src for src in transitive_sources.to_list()]
     proto_include_paths = [
         "-I%s" % p
         for p in _proto_include_paths(
             [f for f in all_sources],
-        )
+        ).to_list()
```
Comment (Yannic) [inline bazel/closure_grpc_web_library.bzl]: I think you could change `_proto_include_paths` to return a list instead. Not sure why it returns a depset...

Comment (factuno-db) [reply bazel/closure_grpc_web_library.bzl]: Done!

**Comment by factuno-db** [comment]
Done!

**Comment by nghialv** [comment]
Thank you. Can we have a merge of this soon?

**Comment by stanley-cheung** [comment]
Apparently, when executing `./bazel-0.27.0-installer-linux-x86_64.sh --user`,

this is the error message:

```
## Build information
   - [Commit](https://github.com/bazelbuild/bazel/commit/c82eb48)
Uncompressing....../home/kbuilder/bin/bazel: relocation error: /home/kbuilder/bin/bazel: symbol _ZTVNSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEEE, version GLIBCXX_3.4.21 not defined in file libstdc++.so.6 with link time reference


[ID: 5297890] Build finished after 815 secs, exit value: 127
```

Could be an environment issue.

**Comment by Yannic** [comment]
Is Kokoro using Ubuntu 14.04? Bazel started building its releases on 16.04 starting with 0.27.0 which could cause issues like that.
https://blog.bazel.build/2019/06/17/bazel-0.27.0.html

**Comment by factuno-db** [comment]
Yep, sounds like Ubuntu 14.04 is the issue. Is there anything I can do?

**Comment by Globegitter** [comment]
There was some discussion here: https://github.com/bazelbuild/bazel/issues/7816#issuecomment-505461522 and it seems the next release should support Ubuntu 14.04 again.

**Comment by Yannic** [comment]
@factuno-db Can we upgrade to / test with Bazel 0.27.1?

**Comment by factuno-db** [comment]
Sure, I've checked daily for new Bazel releases ;-)

**Comment by stanley-cheung** [comment]
Yay tests passed :)

**Pull request merged**